.github/workflows/dependencies.yml

```diff
@@ -37,40 +37,11 @@ jobs:
           echo "::set-output name=dependencies::$(echo "$DEPENDENCIES")"
           shell: bash
 
-      - name: Send Slack notification
-        uses: 8398a7/action-slack@v3
+      - name: Upload dependencies report
+        uses: actions/upload-artifact@v2
         with:
-          status: custom
-          custom_payload: |
-            {
-            	"attachments": [
-            		{
-            			"color": "#3DDC84",
-            			"blocks": [
-            				{
-            					"type": "header",
-            					"text": {
-            						"type": "plain_text",
-            						"text": "Dependency check",
-            					}
-            				},
-            				{
-            					"type": "divider"
-            				},
-            				{
-            					"type": "section",
-            					"text": {
-            						"type": "mrkdwn",
-            						"text": '${{ env.depss }}'.split('%0A').join('\n').split('%0D').join('\r')
-            					}
-            				}
-            			]
-            		}
-            	]
-            }
-        env:
-          SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
-        if: always()
+          name: report
+          path: build/dependencyUpdates/report.txt
 
       - name: Set Job Status
         id: status
```

Comment:
Can not add a comment to below, so adding here. Let's add `Notify slack success` also in order to get notification of each successful dependency check task. You can check the `release.yml` as an example

Reply:
Added 5511e28cc0194c51fa962e52b6f48fb2952c49cf 👍